manufacturing/internals/display_panel/electronics/README.md

```diff
@@ -13,28 +13,22 @@ BEFORE purchasing any parts.**
 
 | Item  | Quantity | Manufacturer  | Part Number        | Price (USD) | Sources[*][ppg]         | Notes |
 | ----  |---------:| --------------| ------------------ | -----------:|:-----------------------:| ----- |
-|**A1** | 1        | RespiraWorks  | PCB                |             | [Rw][a1rw]              | RespiraWorks custom PCB |
-|**A2** | (1)      | RespiraWorks  | PCB_interim_board  |             | [Rw][a2rw]              | RespiraWorks Rev1 interim board **OPTIONAL** |
+|**A1** | 1        | RespiraWorks  | [PCB][a1rw]        |             | Rw                      | Custom main board |
+|**A2** | (1)      | RespiraWorks  | [interim_board][a2rw] |          | Rw                      | Rev1 interim board **OPTIONAL** |
 |**A3** | (1)      | Duracell      | 2016 3V            | 1.67        | [Z][a3amzn]             | CR2016 battery, for interim board **OPTIONAL** |
 |**A4** | 1        | ST Micro      | NUCLEO-L452-RE     | 14.43       | [M][a4mous] [R][a4rs]   | STM32 Nucleo dev board, **DO NOT get the -P** version |
 |**A5** | 2        | Digikey       | X-NUCLEO-IHM03A1   | 10.64       | [K][a5key] [M][a5mous]  | Stepper driver board |
 |**A6** | (1)      | ST            | X-NUCLEO-IHM02A1   | 15.00       | [S][a6st] [R][a6rs]     | Two-axis stepper driver board, **Alternative to 2x`[A5]`**  |
 |**A7** | 1        | Raspberry Pi  | RPI4-MODBP-2GB     | 53.23       | [M][a7mous]             | Raspberry Pi 4 |
 |**A8** | 1        | Pastall       | PT2019120501       | 9.99 / 40   | [Z][a8amzn]             | Heat sink for Rpi |
 |**A9** | 1        | SanDisk       | SDSQXAF-032G-GN6MA | 10.73       | [Z][a9amzn]             | UHS-1 Class 10 micro SD, memory for Raspberry Pi |
-|**A10**| 1        | UPerfect      | -                  | 49.00       | [A][a10ali]             | 7" capacitive touchscreen with speakers |
-|**A11**| 1        | UGREEN        | B0773N757H         | 6.99        | [Z][a11amzn]            | USB-A to micro-USB cable, 1.5ft |
-|**A12**| 1        | Seadream      | 26121600           | 7.00        | [Z][a12amzn]            | microHDMI adapter |
-|**A13**| 1        | C2G           | 56781              | 7.99        | [Z][a13amzn]            | HDMI cable, 1ft |
+|**A10**| 1        | UGREEN        | B0773N757H         | 6.99        | [Z][a10amzn]            | USB-A to micro-USB cable, 1.5ft |
+|**A11**| 1        | CERRXIAN      | B01N5PHSJE         | 6.95        | [Z][a11amzn]            | USB-A to mini-USB cable, 9", right angle |
+|**A12**| 4        | DigiKey       | 732-10435-ND       | 50.82 / 100 | [K][a12key]             | M2.5 standoff, 20mm |
```

Comment:
This needs to be a 22mm standoff, per #1027

- part link https://www.digikey.com/en/products/detail/raf-electronic-hardware/M2117-2545-AL/9836050
- part number M2117-2545-AL

Reply:
Corrected. Added disclaimer in the instructions explaining discrepancy in CAD.